xml/cap_depl_install_production.xml

```diff
@@ -968,37 +999,54 @@ sizing:
 <screen>
 sizing:
   api:
-    count: 2
+    count: 6
+  cc_uploader:
+    count: 3
+  cc_worker:
+    count: 6
   cf_usb:
-    count: 2
+    count: 3
   consul:
     count: 1
   diego_access:
-    count: 2
+    count: 3
   diego_api:
     count: 3
   diego_brain:
     count: 2
   diego_cell:
     count: 3
-  doppler:
-    count: 2
-  etcd:
+  diego_locket:
     count: 3
+  doppler:
+    count: 4
   loggregator:
-    count: 2
+    count: 7
   mysql:
     count: 3
+  mysql_proxy:
+    count: 3
```

Comment:
Same as above; don't scale this yet.

Reply:
Yikes. The whole point of this section is showing customers how to get correct information. I shall note these exceptions.